### PR TITLE
solveset sinh(x), hyperbolic Functions

### LINKED ISSUE #7914: solve(Eq(sin(2*x)*cos(x)+cos(2*x)*sin(x), 1))
Dnot found solution for

sin(2_x)_cos(x)+cos(2_x)_sin(x) == 1

The solution must be:

http://www.wolframalpha.com/input/?i=sin%282*x%29*cos%28x%29%2Bcos%282*x%29*sin%28x%29%3D1

### LINKED ISSUE #9531: solveset(sinh(x)) doesn't returns all solutions
``` python
In [2]: solveset(sinh(x))
Out[2]: {0}
```

### PR DESCRIPTION
**Pull request by Shekharrajak**
Fixes https://github.com/sympy/sympy/issues/9531 , https://github.com/sympy/sympy/issues/9824 , https://github.com/sympy/sympy/issues/10426 and https://github.com/sympy/sympy/issues/9606
also solve an [unsolvable test-case](https://github.com/sympy/sympy/pull/10552/files#diff-85baa04bbf4e1dfd9128782738e45424R727)

```
# complex solution
>>> solveset(sinh(x))
{n⋅ⅈ⋅π | n ∊ ℤ}
# Real solution
>>> solveset(sinh(x), x, S.Reals)
{0}

```

Using PR https://github.com/sympy/sympy/pull/10482

Edit -1
Also fixes https://github.com/sympy/sympy/issues/7914

### CODE REVIEW

**Comment by Shekharrajak** [comment]
Please any Sympy member try to merge `sympy` master branch to `shekharrajak/sympy` master branch. My master branch is not up-to-date so it is not showing any conflict in my local system. May be because I mistakenly changed the my master branch HEAD.

**Comment by Shekharrajak** [comment]
You can see there is no conflict with my base/master branch. 
https://github.com/Shekharrajak/sympy/pull/2

**Comment by jksuom** [comment]
There is probably a conflict between sympy/sympy:master and Shekharrajak/sympy:master. It is important that the branch be merged with the former, not with the latter.

**Comment by Shekharrajak** [comment]
should I do this ? 
https://github.com/Shekharrajak/sympy/compare/master...sympy:master

**Comment by Shekharrajak** [comment]
Finally I changed the HEAD of my master branch.

**Comment by smichr** [comment]
:eyeglasses: See [here for auto-closing syntax](https://help.github.com/articles/closing-issues-via-commit-messages/); what you wrote in the OP ("Fixex #9531 and #9606") doesn't follow the syntax.

File: sympy/solvers/tests/test_solveset.py [hunk 1]
```diff
@@ -709,21 +706,9 @@ def test_solveset_complex_tan():
 
 def test_solve_trig():
     from sympy.abc import n
-    assert solveset_real(sin(x), x) == \
```
Comment (smichr) [inline sympy/solvers/tests/test_solveset.py]: rewrite as I showed previously; none of these tests should be deleted. They should all be modified to show the correct value to the right of the `==`.

Comment (Shekharrajak) [inline sympy/solvers/tests/test_solveset.py]: sorry I forgot to mention these test case are written [here](https://github.com/sympy/sympy/pull/10552/files#diff-85baa04bbf4e1dfd9128782738e45424R737)

**Comment by Shekharrajak** [comment]
I didn't understand why travis build failed in this [test case](https://github.com/Shekharrajak/sympy/commit/b1f06b08ce47d2ebfae27ae55eb646170f2653de#diff-85baa04bbf4e1dfd9128782738e45424R1060) It works fine  and no Assertion Error when I tested locally.

**Comment by hargup** [comment]
@Shekharrajak Your branch has merge conflicts.

**Comment by Shekharrajak** [comment]
@hargup , It is because `solveset_real` and `solveset_complex` is changed now. I have rewritten this PR here https://github.com/sympy/sympy/pull/10733 with some additional changes.

**Comment by hargup** [comment]
>  I have rewritten this PR here #10733 with some additional changes.

Cool, so I'm closing this one.

**Comment by Shekharrajak** [comment]
Okay .

**Pull request closed without merging**